#ifndef BCSETTINGPASSWORDSTYLE_H
#define BCSETTINGPASSWORDSTYLE_H

#include <QDialog>

namespace Ui {
class BCSettingPasswordStyle;
}

class BCSettingPasswordStyle : public QDialog
{
    Q_OBJECT

public:
    explicit BCSettingPasswordStyle(QWidget *parent = 0,bool hiden = true);
    ~BCSettingPasswordStyle();

private slots:
    void on_m_pOKBtn_clicked();

    void on_m_pCancelBtn_clicked();

private:
    Ui::BCSettingPasswordStyle *ui;

    void updatePwd(const QString &pwd);
};

#endif // BCSETTINGPASSWORDSTYLE_H
